rpc/jsoncodec/doc.go

```diff
@@ -1,5 +1,13 @@
-// Copyright 2022 Canonical Ltd.
+// Copyright 2025 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-// Package jsoncodec provides a JSON codec for the rpc package.
+// Package jsoncodec defines the methods for reading and writing RPC messages, enabling serialization and deserialization.
+
+// Key components
+
```

Comment:
I'd work this into the intro paragraph. E.g.,

Package jsoncodec defines the methods for reading and writing RPC messages, including
- the methods for sending and receiving JSON messages.
- the message formats (inMsgV0, inMsgV1, outMsgV0, outMsgV1) for different protocol versions.

This enables serialization and deserialization.

PS What's the connection between RPC messages and JSON messages? (If this is something you think everyone will know, please ignore.)

Reply:
Got it, thanks!